tests/DCPS/FilterExpression/FilterStruct.idl

```diff
@@ -1,5 +1,6 @@
 #include <dds/DdsDcpsInfrastructure.idl>
 
+@topic
```

Comment:
Did the test fail because TBTD didn't have `MarshalTraits`?

Reply:
Yes